src/configurationProvider.ts

```diff
@@ -282,3 +384,14 @@ interface IMainClassOption {
     readonly projectName?: string;
     readonly mainClass: string;
 }
+
+interface IValidationResult {
+    readonly passed: boolean;
+    readonly message?: string;
```

Comment:
message ==> errorMessage?

Reply:
after discussion, we think the neutral `message` name is more appropriate, because message could be for both valid and invalid case.